ISISComputingGroup/ibex_gui
# Ticket 3391: Load and save NICOS scripts

### Description of work

This adds the ability to load scripts into the NICOS queue and save from the queue.

### Ticket

See https://github.com/ISISComputingGroup/IBEX/issues/3391

### Acceptance criteria

To test:
* Go to add a script to the NICOS queue, use the open folder icon to load a script from disk
* Before submitting, change the file and save it back to disk
* Confirm file has changed on disk
* Right click on a queued script, save it to disk
* Confirm the script saves correctly

### Unit tests

UI based, added no tests.

---

#### Code Review


### Final Steps
